src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-list/user-list.component.ts

```diff
@@ -83,8 +83,8 @@ export class UserListComponent implements OnInit {
         this.modalRef.hide();
         this.notificationService.show(
           NotificationType.success,
-          `User "${username}" has been deleted.`,
-          'Delete User'
+          'Delete User',
```

Comment:
Could you please use the wording of task messages here? (Also noted in `hacking.rst`) =>
`Deleted user ${username}`